### PR TITLE
Purge "will" from Client Cfg

### PR DESCRIPTION
**Pull request by Loquacity**
# Description

Plus the occasional drive-by.

# Target branches

Which product version does this PR apply to?

- [ ] Manager-4.0
- [ ] Manager-4.1
- [x] Manager-4.2
- [x] Uyuni

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/clients-rh-cdn.adoc [hunk 1]
```diff
@@ -183,8 +183,8 @@ include::snippets/manual_channels.adoc[]
 [IMPORTANT]
 ====
 For {redhat} 8 clients, add both the Base and AppStream channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+You require packages from both channels.
+If you do not add both channels, you can not create the bootstrap repository, due to missing packages.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-cdn.adoc:187]:
```suggestion
If you do not add both channels, you cannot create the bootstrap repository, due to missing packages.
```

File: modules/client-configuration/pages/clients-rh-rhui.adoc [hunk 2]
```diff
@@ -162,8 +162,8 @@ include::snippets/manual_channels.adoc[]
 [IMPORTANT]
 ====
 For {redhat} 8 clients, add both the Base and AppStream channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+You require packages from both channels.
+If you do not add both channels, you can not create the bootstrap repository, due to missing packages.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh-rhui.adoc:166]:
```suggestion
If you do not add both channels, you cannot create the bootstrap repository, due to missing packages.
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 3]
```diff
@@ -151,8 +151,8 @@ include::snippets/manual_channels.adoc[]
 [IMPORTANT]
 ====
 For {redhat} 8 clients, add both the Base and AppStream channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+You require packages from both channels.
+If you do not add both channels, you can not create the bootstrap repository, due to missing packages.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc:155]:
```suggestion
If you do not add both channels, you cannot create the bootstrap repository, due to missing packages.
```

File: modules/client-configuration/pages/cobbler.adoc [hunk 4]
```diff
@@ -409,9 +408,9 @@ New systems are added to the organization of the administrator who enabled the b
 To change the organization, log in as an Administrator of the required organization, and re-enable the feature.
 ====
 
-When the feature has been enabled, any bare metal system connected to the server network will be automatically added to the organization when it is powered on.
-The process can take a few minutes, and the system will automatically shut down when it is complete.
-The system will now be visible in the menu:Systems[System list].
+When the feature has been enabled, any bare metal system connected to the server network is automatically added to the organization when it is powered on.
+The process can take a few minutes, and the system automatically shutd down when it is complete.
```
Comment (keichwa) [inline modules/client-configuration/pages/cobbler.adoc:412]:
```suggestion
The process can take a few minutes, and the system automatically shuts down when it is complete.
```

File: modules/client-configuration/pages/cobbler.adoc [hunk 5]
```diff
@@ -421,12 +420,12 @@ You can migrate bare metal systems to other organizations if required, using the
 === Provision Bare Metal Systems
 
 Provisioning bare metal systems is similar to provisioning other systems, and can be done using the [guilabel]``Provisioning`` tab.
-However, you will not be able to schedule provisioning, it will happen automatically as soon as the system is configured and powered on.
+However, you can not schedule provisioning, it happens automatically as soon as the system is configured and powered on.
```
Comment (keichwa) [inline modules/client-configuration/pages/cobbler.adoc:423]:
```suggestion
However, you cannot schedule provisioning, it happens automatically as soon as the system is configured and powered on.
```

File: modules/client-configuration/pages/contact-methods-osad.adoc [hunk 6]
```diff
@@ -77,7 +77,7 @@ We recommend you keep the default configuration parameters for all OSAD componen
 If your OSAD clients cannot connect to the server, or if the [systemitem]``jabberd`` service takes a lot of time responding to port 5552, it could be because you have exceeded the open file count.
 
 Every client needs one always-open TCP connection to the server, which consumes a single file handler.
-If the number of file handlers currently open exceeds the maximum number of files that [systemitem]``jabberd`` is allowed to use, [systemitem]``jabberd`` will queue the requests, and refuse connections.
+If the number of file handlers currently open exceeds the maximum number of files that [systemitem]``jabberd`` is allowed to use, [systemitem]``jabberd`` queues the requests, and refuse connections.
```
Comment (keichwa) [inline modules/client-configuration/pages/contact-methods-osad.adoc:80]:
```suggestion
If the number of file handlers currently open exceeds the maximum number of files that [systemitem]``jabberd`` is allowed to use, [systemitem]``jabberd`` queues the requests, and refuses connections.
```
drive-by suggestion

File: modules/client-configuration/pages/registration-bootstrap.adoc [hunk 7]
```diff
@@ -10,7 +10,7 @@ The bootstrap script you create is executed on the client when it is registered,
 There are some parameters contained in the bootstrap script, which ensure the client system can be assigned to its base channel, including activation keys and GPG keys.
 
 It is important that you check the repository information carefully, to ensure it matches the base channel repository.
-If the repository information does not match exactly, the bootstrap script will not be able to download the correct packages.
+If the repository information does not match exactly, the bootstrap script can not download the correct packages.
```
Comment (keichwa) [inline modules/client-configuration/pages/registration-bootstrap.adoc:13]:
```suggestion
If the repository information does not match exactly, the bootstrap script cannot download the correct packages.
```

File: modules/client-configuration/pages/snippets/trust_gpg.adoc [hunk 8]
```diff
@@ -1,6 +1,6 @@
 By default, some operating systems do not trust the GPG key for the {productname} client tools.
 The clients can be successfully bootstrapped without the GPG key being trusted.
-However, you will not be able to install new client tool packages or update them until the keys are trusted.
+However, can not install new client tool packages or update them until the keys are trusted.
```
Comment (keichwa) [inline modules/client-configuration/pages/snippets/trust_gpg.adoc:3]:
```suggestion
However, you cannot install new client tool packages or update them until the keys are trusted.
```

File: modules/client-configuration/pages/vhm-caasp.adoc [hunk 9]
```diff
@@ -101,7 +101,7 @@ You can use a single management node for multiple clusters, as long as the clust
 
 [NOTE]
 ====
-You will not be able to use the management node until the highstate has been completed.
+You can not use the management node until the highstate has been completed.
```
Comment (keichwa) [inline modules/client-configuration/pages/vhm-caasp.adoc:104]:
```suggestion
You cannot use the management node until the highstate has been completed.
```

File: modules/client-configuration/pages/virtualization.adoc [hunk 10]
```diff
@@ -124,14 +124,14 @@ The [guimenu]``Schedule`` panel helps configuring when the edit should take plac
 
 [NOTE]
 ====
-If a guest contains one or more disks or network interfaces not recognized by {susemgr}, you will not be able to edit the configuration.
+If a guest contains one or more disks or network interfaces not recognized by {susemgr}, you can not edit the configuration.
```
Comment (keichwa) [inline modules/client-configuration/pages/virtualization.adoc:127]:
```suggestion
If a guest contains one or more disks or network interfaces not recognized by {susemgr}, you cannot edit the configuration.
```

**Review by keichwa — APPROVED** [review]

**Review by jcayouette — APPROVED** [review]
Looks good thanks!

File: modules/client-configuration/pages/activation-keys.adoc [hunk 11]
```diff
@@ -255,7 +255,7 @@ xref:client-configuration:contact-methods-intro.adoc[].
 +
 .Changing the Default Activation Key
 WARNING: Only one universal default activation key can be defined per organization.
-If a universal key already exists for this organization, you will unset the currently used universal key by activating the check box.
+If a universal key already exists for this organization, you cam unset the currently used universal key by activating the check box.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/client-configuration/pages/activation-keys.adoc:258]: :joy:

File: modules/client-configuration/pages/clients-centos.adoc [hunk 12]
```diff
@@ -77,8 +77,8 @@ endif::[]
 [IMPORTANT]
 ====
 For {centos} 8 clients, add both the Base and AppStream channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
+You require packages from both channels.
+If you do not add both channels, you can not create the bootstrap repository, due to missing packages.
```
[the remark this answers is not in the document]
Comment (Loquacity) [reply modules/client-configuration/pages/clients-centos.adoc:81]: Argh, my Australian is showing again!
